modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java

```diff
@@ -694,7 +694,10 @@ public void updateCodegenPropertyEnum(final CodegenProperty property) {
         }
         final String prefix = toEnumName(property);
         for (Map<String, Object> enumVar : enumVars) {
-            enumVar.put("name", prefix + enumVar.get("name"));
+            if (!enumVar.containsKey("_isPrefixed")) {
+                enumVar.put("name", prefix + enumVar.get("name"));
+                enumVar.put("_isPrefixed", true);
```

Comment:
An idea for an alternative: Test if `enumVar.get("name")` already starts with `prefix`. I guess it needs also to be not equal to `prefix` to support prefixing `type MyType = MyType`. Would this actually work and make storing `_isPrefixed` obsolete?

Reply:
Thought of that as well, but I'd like to keep the prefix consistent.

Reply:
Okay